kernel/src/returncode.rs

```diff
@@ -4,7 +4,7 @@
 //!  Author: Philip Levis <pal@cs.stanford.edu>
 //!  Date: Dec 22, 2016
 
-#[derive(Clone, Copy, PartialEq)]
+#[derive(Clone, Copy, Debug, PartialEq)]
```

Comment:
It's perfectly appropriate to add - though a fine argument that it doesn't fit in this commit